app/src/main.ts

```diff
@@ -159,6 +169,19 @@ if (shouldQuit) {
         });
       });
     }
+    if (!appArgs.disableOldBuildWarning) {
+      if (
```

Comment:
Pls merge into a single `if` statement for less indentation and more readable control flow.